diff --git a/app/Filament/Resources/TeacherAssignmentResource.php b/app/Filament/Resources/TeacherAssignmentResource.php
index ff3242a..4aa9111 100644
--- a/app/Filament/Resources/TeacherAssignmentResource.php
+++ b/app/Filament/Resources/TeacherAssignmentResource.php
@@ -122,11 +122,11 @@ public static function form(Form $form): Form
 
                     fn($record) =>
 
-                    $record->gradeSubject->grade->title
+                    $record->appGradeSubject->grade->title
 
                         . ' - ' .
 
-                        $record->gradeSubject->subject->title
+                        $record->appGradeSubject->subject->title
 
                         . ' - ' .
 
@@ -207,7 +207,7 @@ public static function table(Table $table): Table
 
 
                 Tables\Columns\TextColumn::make(
-                    'book.gradeSubject.grade.title'
+                    'book.appGradeSubject.grade.title'
                 )
 
                     ->label('پایه')
@@ -218,7 +218,7 @@ public static function table(Table $table): Table
 
 
                 Tables\Columns\TextColumn::make(
-                    'book.gradeSubject.subject.title'
+                    'book.appGradeSubject.subject.title'
                 )
 
                     ->label('درس')
